db/changelog/v-1/sqlRequests/agent-status.csv

```diff
@@ -1,10 +1,10 @@
-id;time;state;agent_id
-1;"2021-01-01 00:00:00";"IDLE";1
-2;"2021-01-01 00:00:15";"IDLE";1
-3;"2021-01-01 00:00:30";"BUSY";1
-4;"2021-01-01 00:00:45";"BUSY";1
-5;"2021-01-01 00:01:00";"BUSY";1
-6;"2021-01-01 00:01:15";"BUSY";1
-7;"2021-01-01 00:01:30";"FINISHED";1
-8;"2021-01-01 00:01:45";"IDLE";1
-12;"2021-01-01 01:00:00";"BUSY";2
\ No newline at end of file
+id;start_time;end_time;state;agent_id
+1;"2021-01-01 00:00:00";"2021-01-01 00:00:00";"IDLE";1
```

Comment:
I believe end time should be at least 15 seconds further in time than start; better make these intervals different throughout test data